en/simulation/ignition_gazebo.md

```diff
@@ -56,12 +56,114 @@ In order to run the simulation without running the ignition gazebo gui, one can
 HEADLESS=1 make px4_sitl gz_x500
 ```
 
-In order to increase the verbose output, `VERBOSE_SIM=1` can be used:
+## Usage/Configuration Options
+
+The startup pipeline allows for highly flexible configuration.
+In particular, it is possible to:
+
+- Start a new Ignition simulation with an arbitrary world or attach to an already running simulation.
+- Add a new vehicle to Ignition or link a new PX4 instance to an existing one.
+
+These scenarios are managed by setting the appropriate environment variables.
+
+### Syntax
+
+The startup syntax takes the form:
 
 ```bash
-VERBOSE_SIM=1 make px4_sitl gz_x500
+ARGS ./build/px4_sitl_default/bin/px4
 ```
 
+where `ARGS` is a list of environment variables including:
+
+- `PX4_SYS_AUTOSTART` (_mandatory_):
+  Sets the [airframe autostart id](../dev_airframes/adding_a_new_frame.md) of the PX4 airframe to start.
+  Only `4001` (x500 quadcopter) is currently supported.
+
+- `PX4_GZ_WORLD` (_optional_):
+  Sets the Ignition world file for a new simulation.
+  If it is not given, then [default](https://github.com/PX4/PX4-Autopilot/blob/main/Tools/simulation/gz/worlds/default.sdf) is used.
+  This variable is ignored if an existing simulation is already running.
+
+- `PX4_GZ_MODEL_NAME`:
+  Sets the name of an **existing** model in the gazebo simulation.
+  When provided, the startup script tries to bind a new PX4 instance to the Ignition resource matching exactly that name.
+  It is mutually exclusive with `PX4_GZ_MODEL`.
+
+- `PX4_GZ_MODEL`:
+  Sets the name of a new Ignition model to be spawned in the simulator.
+  When provided, the startup script looks for a model in the Ignition resource path that matches the given variable, spawns it and binds a new PX4 instance to it.
+  It is mutually exclusive with `PX4_GZ_MODEL_NAME`.
+
+- `PX4_GZ_MODEL_POSE` (_optional_):
+  Sets the spawning position of the model when `PX4_GZ_MODEL` is adopted.
+  When provided, then startup script spawns the model at the given position.
+  If omitted, the origin `[0,0,0]` is used.
+
+The PX4 Ignition worlds and and models databases [can be found on Github here](https://github.com/PX4/PX4-Autopilot/tree/main/Tools/simulation/gz).
+They are added to the Ignition search `PATH` by [gazebo_env.sh.in](https://github.com/PX4/PX4-Autopilot/blob/main/src/modules/simulation/gz_bridge/gazebo_env.sh.in) during the simulation startup phase.
+
+:::note
+`gazebo_env.sh.in` is compiled and made available in `$PX4_DIR/build/px4_sitl/rootfs/gazebo_env.sh`
+:::
+
+### Examples
+
+Here are some examples of the different scenarios covered above.
+
+1. **Start simulator + default world + spawn vehicle at default location**
+
+   ```sh
+   PX4_SYS_AUTOSTART=4001 PX4_GZ_MODEL=x500 ./build/px4_sitl_default/bin/px4
+   ```
+
+2. **Start simulator + default world + spawn vehicle at custom location**
+
+   ```sh
+   PX4_SYS_AUTOSTART=4001 PX4_GZ_MODEL_POSE="0,0" PX4_GZ_MODEL=x500 ./build/px4_sitl_default/bin/px4
```

Comment:
This is 0, 0 but the default option listed above is 0,0,0. So
- Might be worth showing a different option.
- What is the origin/co-ordinate system relative to?
- What are the units?

Reply:
Thanks for bringing this up, I review the code and

- pose is position(x,y,z)+orientation(roll,pitch,yaw) [metres, radians],
- if less than 6 values are given, zeros are added,
- the origin is the zero pose in Ignition.